Improvements/discount.js

```diff
@@ -0,0 +1,7 @@
+function calculateDiscount(totalAmount) {
+  let discount;
+
+  return totalAmount > 100
+    ? (discount = totalAmount * 0.1)
+    : (discount = totalAmount * 0.05);
+}
```

Comment:
Is there a reason you created the variable?

Reply:
I did know that It can work without that variable, but even after removing it works perfectly